src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/config/NotificationServiceConfiguration.java

```diff
@@ -29,4 +29,5 @@
     private String contestedConsentOrderApproved;
     private String contestedConsentGeneralOrder;
     private String contestedConsentOrderNotApproved;
+    private String contestedGeneralApplicationReferToJudge;
```

Comment:
I have added a Test for this in my PR - in case for whatever reason values are changed